<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Optimisation:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="optimisationComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="newOptimisationButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>New Optimisation....</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptimisations">
    <property name="title">
     <string>Optimisations</string>
    </property>
    <widget class="QMenu" name="menuSwitch_Optimisation">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Switch Optimisation</string>
     </property>
     <addaction name="separator"/>
    </widget>
    <addaction name="actionNewOptimisation"/>
    <addaction name="menuSwitch_Optimisation"/>
   </widget>
   <widget class="QMenu" name="menuLog">
    <property name="title">
     <string>Log</string>
    </property>
    <addaction name="actionShowLog"/>
   </widget>
   <widget class="QMenu" name="menuMesher">
    <property name="title">
     <string>Mesher</string>
    </property>
    <addaction name="actionRunMesher"/>
   </widget>
   <addaction name="menuOptimisations"/>
   <addaction name="menuMesher"/>
   <addaction name="menuLog"/>
  </widget>
  <action name="actionNewOptimisation">
   <property name="text">
    <string>New...</string>
   </property>
   <property name="toolTip">
    <string>New Optimisation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionShowLog">
   <property name="text">
    <string>Show...</string>
   </property>
   <property name="toolTip">
    <string>ShowLog</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionRunMesher">
   <property name="text">
    <string>Run Mesher...</string>
   </property>
   <property name="toolTip">
    <string>Run Mesher...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
